public/v4/apps/mayan-edms.yml

```diff
@@ -0,0 +1,158 @@
+captainVersion: 4
+services:
+    # Mayan EDMS
+    $$cap_appname:
+        depends_on:
+            - $$cap_appname-db
+            - $$cap_appname-redis
+        image: mayanedms/mayanedms:$$cap_app_version
+        restart: always
+        environment:
+            MAYAN_CELERY_BROKER_URL: 'redis://:$$cap_redis_pass@srv-captain--$$cap_appname-redis:6379/0'
+            MAYAN_CELERY_RESULT_BACKEND: 'redis://:$$cap_redis_pass@srv-captain--$$cap_appname-redis:6379/1'
+            MAYAN_DATABASES: "{'default':{'ENGINE':'django.db.backends.postgresql','NAME':'$$cap_dbname','PASSWORD':'$$cap_dbpass','USER':'$$cap_dbuser','HOST':'srv-captain--$$cap_appname-db'}}"
+            MAYAN_LOCK_MANAGER_BACKEND: 'mayan.apps.lock_manager.backends.redis_lock.RedisLock'
+            MAYAN_LOCK_MANAGER_BACKEND_ARGUMENTS: "{'redis_url':'redis://:$$cap_redis_pass@srv-captain--$$cap_appname-redis:6379/2'}"
+            MAYAN_SEARCH_BACKEND: 'mayan.apps.dynamic_search.backends.whoosh.WhooshSearchBackend'
+            MAYAN_APT_INSTALLS: '$$cap_ocr_langs'
+            MAYAN_DOCKER_WAIT: srv-captain--$$cap_appname-db:5432 srv-captain--$$cap_appname-redis:6379
+            MAYAN_DOCUMENTS_LANGUAGE: $$cap_default_document_lang
+            MAYAN_TIME_ZONE: $$cap_timezone
+            MAYAN_AUTOADMIN_EMAIL: $$cap_email
+            MAYAN_AUTOADMIN_PASSWORD: $$cap_password
+            MAYAN_AUTOADMIN_USERNAME: $$cap_username
+            MAYAN_COMMON_PROJECT_TITLE: $$cap_project_title
+        volumes:
+            - $$cap_appname-data:/var/lib/mayan
+        caproverExtra:
+            containerHttpPort: '8000'
+
+    # Redis
+    $$cap_appname-redis:
+        volumes:
+            - $$cap_appname-redis-data:/data
+        restart: always
+        caproverExtra:
+            dockerfileLines:
+                - FROM redis:$$cap_redis_version
+                - CMD exec redis-server --appendonly 'no' --databases '3' --maxmemory 100mb --maxclients '500' --maxmemory-policy allkeys-lru --save '' --requirepass $$cap_redis_pass
+            notExposeAsWebApp: 'true'
+
+    #PostgreSQL
+    $$cap_appname-db:
+        image: postgres:$$cap_postgres_version
+        volumes:
+            - $$cap_appname-db-data:/var/lib/postgresql/data
+        restart: always
+        command:
```

Comment:
`command` is ignored by CapRover. If it's needed convert to dockerfileLines, otherwise, let's remove it

Reply:
I was trying to be faithful to the recommended docker-compose.yml but it works fine with the postgres defaults, so I'll leave it like that and don't use DockerfileLines. Thanks